packages/kbn-es/src/cli_commands/serverless.ts

```diff
@@ -66,20 +70,43 @@ export const serverless: Command = {
     });
     const reportTime = getTimeReporter(log, 'scripts/es serverless');
 
+    // replacing --serverless with --projectType when flag is passed from 'scripts/es'
```

Comment:
Similar to the comment in `cli.ts`, I'd suggest to provide some extra information:
```suggestion
    // replacing --serverless with --projectType when flag is passed from 'scripts/es'
    // `es --serverless=<projectType>` is just a shortcut for
    // `es serverless --projectType=<projectType>`
```